src/app/modules/admin/admin.routes.ts

```diff
@@ -1,14 +1,18 @@
-import {Routes} from "@angular/router";
-
-const routes: Routes = [
-  {
-    path: 'customers',
-    loadComponent: () => import('./pages/customer/customer.page').then((m) => m.CustomerPage),
-  },
  {
+import { Routes } from "@angular/router";
+
+const routes: Routes = [
+  {
+    path: 'customers',
+    loadComponent: () => import('./pages/customer/customer.page').then((m) => m.CustomerPage),
+  },
+  {
     path: 'plans',
-    loadComponent: () => import('./pages/plans/plans.page').then( m => m.PlansPage)
+    loadComponent: () => import('./pages/plans/plans.page').then(m => m.PlansPage)
+  },
+  {
+    path: 'inventario',
```

Comment:
```suggestion
    path: 'inventory',
```
Mantener la misma naturaleza en toda la aplicacion, si los otros nombres estan en inglés, dejarlos todos en inglés.